.clang-tidy

```diff
@@ -11,6 +11,7 @@ Checks:          '-*,
                   modernize-make-unique,
                   modernize-avoid-bind,
                   misc-unused-parameters,
+		  readability-braces-around-statements,
```

Comment:
Can you fix the formatting of this so it lines up with the other clang-tidy checks?